packages/lab/src/combo-box-next/ComboBoxNext.tsx

```diff
@@ -45,36 +39,40 @@ export interface ComboBoxNextProps<T>
    * Additional props for the portal.
    */
   PortalProps?: UseComboBoxPortalProps;
+  /* CONTROLLED PROPS*/
   /**
-   * If `true`, the component will be disabled.
+   * Controlled prop. Controls the value for the Combo Box input.
+   */
+  value?: string;
+  /**
+   * Controlled prop. Controls the Highlighted item in the Combo Box list.
    */
-  disabled: boolean;
-  /* Highlighted index for when the list is controlled. */
   highlightedItem?: string;
-  /* Selected value for when the list is controlled. */
+  /**
+   * Controlled prop. Controls the Selected value in the Combo Box list.
+   */
   selected?: string;
-  /* Initial selected value for when the list is controlled. */
-  defaultSelected?: string;
+  /* UNCONTROLLED PROPS*/
```

Comment:
this potentially makes it look like disabled, variant etc are uncontrolled props

Reply:
yeah, i thought that too and wasn't sure if it was best to leave it or remove it... i think it doesn't add that much now everything has descriptions, will remove it :)